docs_user/modules/proc_adopting-telemetry-services.adoc

```diff
@@ -14,71 +14,87 @@ This guide also assumes that:
-      secret: osp-secret
-      sgCoreImage: quay.io/infrawatch/sg-core:v5.1.1
-endif::[]
-ifeval::["{build}" == "downstream"]
-      centralImage: registry.redhat.io/rhosp-dev-preview/openstack-ceilometer-central-rhel9:18.0
-      computeImage: registry.redhat.io/rhosp-dev-preview/openstack-ceilometer-compute-rhel9:18.0
-      customServiceConfig: |
-        [DEFAULT]
-        debug=true
-      ipmiImage: registry.redhat.io/rhosp-dev-preview/openstack-ceilometer-ipmi-rhel9:18.0
-      nodeExporterImage: quay.io/prometheus/node-exporter:v1.5.0
-      notificationImage: registry.redhat.io/rhosp-dev-preview/openstack-ceilometer-notification-rhel9:18.0
-      secret: osp-secret
-      sgCoreImage: quay.io/infrawatch/sg-core:v5.1.1
-endif::[]
+  channel: development
+  installPlanApproval: Automatic
+  name: cluster-observability-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
 EOF
 ----
 
-. Optional: If you previously backed up your {OpenStackShort} services configuration file from the old environment, you can use os-diff to compare and make sure the configuration is correct. This will produce the difference between both ini configuration files:
+* Wait for the installation to succeed
 +
 ----
-os-diff diff /tmp/collect_tripleo_configs/ceilometer/etc/ceilometer/ceilometer.conf ceilometer_patch.yaml --crd
+oc wait --for jsonpath="{.status.phase}"=Succeeded csv --namespace=openshift-operators -l operators.coreos.com/cluster-observability-operator.openshift-operators
 ----
+
+* Enable metrics storage backend
 +
-For more information, see xref:reviewing-the-openstack-control-plane-configuration_{context}[Reviewing the {rhos_prev_long} control plane configuration].
+----
+oc patch openstackcontrolplane openstack --type=merge --patch '
+spec:
+  telemetry:
+    template:
+      metricStorage:
+        enabled: true
+        monitoringStack:
+          alertingEnabled: true
+          scrapeInterval: 30s
+          storage:
+            strategy: persistent
+            retention: 24h
+            persistent:
+              pvcStorageRequest: 20G
+'
+----
 
-. Patch the `OpenStackControlPlane` CR to deploy Ceilometer services:
+* Verify that `alertmanager` and `prometheus` pods are available
 +
 ----
-oc patch openstackcontrolplane openstack --type=merge --patch-file ceilometer_patch.yaml
+oc get pods -l alertmanager=metric-storage
+NAME                            READY   STATUS    RESTARTS   AGE
+alertmanager-metric-storage-0   2/2     Running   0          46s
+alertmanager-metric-storage-1   2/2     Running   0          46s
+
+oc get pods -l prometheus=metric-storage
+NAME                          READY   STATUS    RESTARTS   AGE
+prometheus-metric-storage-0   3/3     Running   0          46s
 ----
 
-.Verification
+.Procedure
 
-. Inspect the resulting Ceilometer pods:
+* Patch the `OpenStackControlPlane` CR to deploy Ceilometer services:
+// TODO(jistr): There are still some quay.io images in the downstream build.
```

Comment:
This comment is not valid anymore, quay.io images were removed from downstream build.